### ISSUE TITLE
Change size of preview/edit pane

### ISSUE THREAD

**Issue by Qtechnetita** [Zimbra-Community/owncloud-zimlet#210]
Congratulations for the great job done with this zimlet and your support!
Reading issues I understood that resizing preview/edit pane dynamically is not so easy. Instead,
is there a way in the actual release to edit or preview document in a separate tab/window in order to maximize it and get all available windows space to edit? (or a plan to add the feature in a future version)
Thanks

**Comment by barrydegraaff** [comment]
Yeah, it is definitively something that needs to work better. I want all panes to be resizable. But I guess we do need some sort of minimal resolution. Since there really is no point on support 800x600px display sizes, and people seem to also expect that.

Complicating factor is Zimbra outdated UI framework, so I either need to integrate further into it, or less. 

Anyway. It is something that annoys me some times, so it will probably be done. :-p I do not have an ETA though.

**Comment by tuxcrafter** [comment]
Hello Barry, I been getting feature request for resizable columns for the owncloud zimplet as well, would you be able to take a look and see if it is doable to make the columns and preview plane resizable. Thank you!
